update docs for lark support bot (#22425)
pingcap/docs: tidb-cloud/connected-lark-ticket-creation.md

## Create a support ticket

1. In the **PingCAP Support Group** of Lark, mention `@PingCAP Support Bot` and describe the problem in a message. Then, the bot will send you an ephemeral card message that is visible only to you.

    ![lark-ticket-creation-1](/media/tidb-cloud/connected-lark-ticket-creation-1.png)

2. In the card, fill in the **Reporter** field with your email address used in the [TiDB Cloud console](https://tidbcloud.com/), fill in other fields according to the actual problem, and then click **Submit** to create the ticket.

    ![lark-ticket-creation-2](/media/tidb-cloud/connected-lark-ticket-creation-2.png)

After the ticket is created, the bot replies with the ticket link in the thread. You can click the link to view ticket details in the [PingCAP Help Center](https://tidb.support.pingcap.com/servicedesk/customer/portals).

![lark-ticket-creation-3](/media/tidb-cloud/connected-lark-ticket-creation-3.png)

![lark-ticket-creation-4](/media/tidb-cloud/connected-lark-ticket-creation-4.png)

## Subscribe to ticket updates

> For customers in the **Enterprise** support plan, you can also subscribe to ticket updates in Lark. When a support engineer responds to a ticket, an update message is posted in the **PingCAP Support Group** of Lark. The message includes the ticket title, ticket link, and the latest comment content.

Reviewer: ```suggestion
In the **PingCAP Support Group** of Lark, after you [create a support ticket](#create-a-support-ticket), you can receive ticket updates in Lark directly. When a support engineer responds to a ticket, an update message is posted in the **PingCAP Support Group** of Lark. The message includes the ticket title, ticket link, and the latest comment content.
```
